guides/source/form_helpers.md

```diff
@@ -290,95 +334,95 @@ so:
 <% end %>
 ```
 
-Which produces the following output:
+The above will produces the following output:
 
 ```html
 <form action="/people" accept-charset="UTF-8" method="post">
-  <input type="hidden" name="authenticity_token" value="bL13x72pldyDD8bgtkjKQakJCpd4A8JdXGbfksxBDHdf1uC0kCMqe2tvVdUYfidJt0fj3ihC4NxiVHv8GVYxJA==" />
+  <input type="hidden" name="authenticity_token" value="..." autocomplete="off" />
```

Comment:
This isn't a big deal, but the other examples are including the non-redacted authenticity token.

Reply:
That's true. I added it back for the sake of consistency.